addons/event/static/src/client_action/event_barcode.js

```diff
@@ -118,11 +118,16 @@ export class EventScanView extends Component {
 
     onClickSelectAttendee() {
         if (this.isMultiEvent) {
-            this.actionService.doAction("event.event_registration_action");
+            this.actionService.doAction("event.event_registration_action", {
```

Comment:
As explained, for master we can create new views specific for barcode that do not depend on context.

Reply:
And by new views, I mean new actions :roll_eyes:

Reply:
Helloo @tde-banana-odoo 

I understood about the views (that we can solve this issue by making a new view). But could you please elaborate about actions (here too we have to add context, right.. then what's the difference)?  

Thanks :blush: 

PS: These actions are specifically created for the barcode view (when user comes via registration desk).